src/components/saveSelector.html

```diff
@@ -38,6 +38,7 @@
     <div class="trainer-card-container mb-3 col-lg-4 col-md-6 col-sm-12 xol-xs-12">
         <div class="trainer-card clickable card font-weight-bold">
             <div class="card-body">
+                <div class="position-absolute small trainer-id" style="top: 0; right: 5px;"></div>
```

Comment:
I think we want something in front of the number. Maybe just a #

Reply:
Updated